client/lib/plans/constants.js

```diff
@@ -386,12 +395,21 @@ export const getPlanObject = planName => {
 	return objectPlan;
 };
 
-export const getPlanFeaturesObject = planName => {
-	const planFeaturesList = plansList[ planName ].getFeatures();
+/**
+ * Returns the features list object with keys as the feature constants and values as the feature objects.
+ * Example: { [ FEATURE_XY ] : { getTitle: () => i18n.translate( 'Some title' ) } }
+ *
+ * @param   {Array}  planFeaturesConstantList List of plan features constants (ie: [ FEATURE_XY, FEATURE_Z ])
+ * @returns {Object} Features list object
+ */
+export const getPlanFeaturesConstantObj = ( planFeaturesConstantList ) => {
```

Comment:
Do we need `getPlanFeaturesConstantObj`? It doesn't seem like it would be reused